WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -548,7 +548,9 @@ private extension StorePickerViewController {
         guard ServiceLocator.stores.isAuthenticated else {
             return
         }
-        ABTest.start()
+        Task { @MainActor in
+            await ABTest.start()
+        }
```

Comment:
❓ I thought we already started ABTest in `AppDelegate`, why do we need to start it again here? Will this make a difference?

Reply:
Since this code was here before I worked on the A/A test, just my guess - it's trying to refresh the A/B experiment assignments after the user is logged in, in case some assignments are targeted to specific users based on their user ID. I think the name is a bit misleading though, since it's more like "refresh" in this case.